networking/test/Test/NodeSpec.hs

```diff
@@ -46,7 +46,7 @@ import           Test.Util (HeavyParcel (..), Parcel (..), Payload (..),
 spec :: Spec
 spec = describe "Node" $ modifyMaxSuccess (const 50) $ do
 
-    logTrace <- runIO $ wsetupLogging (defaultTestConfiguration Debug) "nodespec"
+    logTrace <- runIO $ wsetupLogging "test" (defaultTestConfiguration Debug) "nodespec"
```

Comment:
`wsetupLogging "test" == walletSetupTestLogging`? Not sure what `w` is for here.

Reply:
it's a different `setupLogging` yielding a `Trace`. 
historically was named after `log-warper`.

Reply:
or `setupTestTrace`?